packages/eslint-plugin/src/rules/import-playwright-a11y.cjs

```diff
@@ -15,13 +15,18 @@ module.exports = {
   create: (context) => ({
     ImportSpecifier(node) {
       // allow the fixture itself to import from Playwright directly
-      if (context.filename.endsWith("/playwright-axe.ts")) return;
+      if (context.filename.endsWith("/playwright/a11y.ts")) return;
 
       const hasFixture = ["expect", "test"].includes(node.imported.name);
       if (!hasFixture) return;
 
       // type check that node.parent.source.value exists
-      if (!("source" in node.parent) || !("value" in node.parent.source)) {
+      if (
+        !("source" in node.parent) ||
+        !node.parent.source ||
```

Comment:
this one is included in the next line, as "value" could not exist in an undefined source
```suggestion
```